protocol/lib/big_math.go

```diff
@@ -45,7 +45,8 @@ func RatPow10(exponent int32) *big.Rat {
 	return result
 }
 
-// BigIntMulPpm takes a `big.Int` and returns the result of `input * ppm / 1_000_000`.
+// BigIntMulPpm takes a `big.Int` and returns the result of `input * ppm / 1_000_000`. This method rounds towards
+// zero.
```

Comment:
I think this rounds towards neg infinity, since input can be negative. Could you update comment/add unit test accordingly?

Reply:
You're right! Updating comment, adding test cases.